Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG UBUNTU_VERSION=16.04
+FROM ubuntu:$UBUNTU_VERSION
+
+RUN set -x \
+ && apt update \
+ && apt install -y curl build-essential \
```

Comment:
This will build the program with GCC, which feels wrong ;), but it seems to be the compiler used for package from apt.llvm.org so that ensures compatibility.